package.json

```diff
@@ -6,7 +6,8 @@
   "types": "dist/index.d.ts",
   "bin": {
     "ts-node": "dist/bin.js",
-    "ts-script": "dist/script.js"
+    "ts-script": "dist/script.js",
+    "ts-node-transpile-only": "dist/transpile.js"
```

Comment:
Based on https://github.com/TypeStrong/ts-node/pull/948#discussion_r370911070 I think we need to rename to `dist/bin-script.js` and `dist/bin-transpile.js`.  Once that's done I can take a stab at merging and publishing a new release.